_pkgdown.yml

```diff
@@ -0,0 +1,4 @@
+url: ~
+template:
+  bootstrap: 5
+
```

Comment:
If the colors (untested) seem off, please feel free to adjust them. I tried to take them from links and grey background in https://leafletjs.com/ .

```suggestion
url: https://rstudio.github.io/leaflet/

development:
  mode: auto

template:
  package: tidytemplate
  bootstrap: 5
  bslib:
    primary: "#199900"
    navbar-background: "#f8f8f8"
  trailing_slash_redirect: true
```

Reply:
You can always readjust css later if you want, but the green makes it look more unique
![Capture d’écran, le 2024-02-16 à 10 03 10 a m](https://github.com/rstudio/leaflet/assets/52606734/6fb6bf46-5b9b-4500-ad80-b90bebcee43c)